ui/components/component-library/skeleton/skeleton.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  BackgroundColor,
+  BlockSize,
+  Display,
+  FlexDirection,
+  BorderRadius,
+  TextVariant,
+  AlignItems,
+} from '../../../helpers/constants/design-system';
+import { Box, Button, ButtonVariant, Text } from '..';
+import README from './README.mdx';
+import { Skeleton } from './skeleton';
+
+const meta: Meta<typeof Skeleton> = {
+  title: 'Components/ComponentLibrary/Skeleton',
+  component: Skeleton,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    className: {
+      control: 'text',
+    },
+  },
+  args: {
+    className: '',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Skeleton>;
+
+export const DefaultStory: Story = {
+  args: {
+    height: 32,
+    width: 300,
+  },
+};
+
+DefaultStory.storyName = 'Default';
+
+export const WidthHeight: Story = {
+  render: () => (
+    <Box display={Display.Flex} flexDirection={FlexDirection.Column} gap={4}>
+      <Skeleton height={32} width={300} />
+      <Skeleton height={16} width={250} />
+      <Skeleton height={16} width={250} />
+    </Box>
+  ),
+};
+
+export const Children: Story = {
+  render: () => (
+    <Skeleton
+      display={Display.Flex}
+      flexDirection={FlexDirection.Column}
+      gap={4}
+      backgroundColor={BackgroundColor.backgroundAlternative}
+      borderRadius={BorderRadius.LG}
+      padding={4}
+    >
+      <Skeleton height={32} width="100%" />
+      <Skeleton height={16} width="95%" />
+      <Skeleton height={16} width="95%" />
+    </Skeleton>
+  ),
+};
```

Comment:
This story should be fixed when this PR gets merged https://github.com/MetaMask/metamask-extension/pull/27783